### PR TITLE
[13.0][MIG] agreement

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa TT27165
ping @carlosdauden @CarlosRoca13 
Superseeds PR https://github.com/OCA/contract/pull/578 without name changed.

### CODE REVIEW

**Comment by sergio-teruel** [comment]
Runbot display fail but is runing... why??

**Comment by pedrobaeza** [comment]
See the video to know how to get to the problem:

https://user-images.githubusercontent.com/7165771/107888848-3471d500-6f0f-11eb-8231-9b3901b825ac.mp4

**Review by carlosdauden — APPROVED** [review]

File: agreement/security/agreement_security.xml [hunk 1]
```diff
@@ -1,26 +1,22 @@
...
-    <field name="name">Use agreement template</field>
-    <field name="category_id" ref="base.module_category_hidden"/>
-</record>
-
+    <record id="agreement_rule" model="ir.rule">
+        <field name="name">Agreement multi-company</field>
+        <field name="model_id" ref="model_agreement" />
+        <field
+            name="domain_force"
+        >['|', ('company_id', '=', False), ('company_id', 'child_of', [user.company_id.id])]</field>
```
Comment (pedrobaeza) [inline agreement/security/agreement_security.xml]: This rule must be transformed and migration scripts should be added. See in https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-13.0 the line about that.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
It seems the migration commit contains black/prettier formatting. Can you please isolate that one so it's not contaminated with it and the review is done more accurately?

**Review by pedrobaeza — APPROVED** [review]

**Review by marcelsavegnago — APPROVED** [review]

**Pull request merged**